es_ES/cookbook/images/fading-in-images.md

```diff
@@ -0,0 +1,124 @@
+---
+layout: page
+title: "Efecto “Fade in” en imágenes con un placeholder"
+permalink: /cookbook/images/fading-in-images/
+---
+
+Al mostrar imágenes usando el widget `Image` predeterminado, es posible que notes que simplemente aparecen en la pantalla a medida que se cargan. Esto puede parecer visualmente molesto para tus usuarios.
+
+En vez de eso, ¿no sería bueno si pudieras mostrar un placeholder al principio, y 
+las imágenes se desvanecerían a medida que se cargan? ¡Podemos usar el Widget [`FadeInImage`](https://docs.flutter.io/flutter/widgets/FadeInImage-class.html) 
+ empaquetado con Flutter exactamente para este propósito!
+
+`FadeInImage` funciona con imágenes de cualquier tipo: en memoria, recursos locales o imágenes de Internet.
+
+## En Memoria
+
+En este ejemplo, usaremos el paquete [transparent_image](https://pub.dartlang.org/packages/transparent_image) para un simple placeholder transparente.
+
+<!-- skip -->
+```dart
+FadeInImage.memoryNetwork(
+  placeholder: kTransparentImage,
+  image: 'https://github.com/flutter/website/blob/master/_includes/code/layout/lakes/images/lake.jpg?raw=true',
+);
+```
+
+### Ejemplo completo
+
+```dart
+import 'package:flutter/material.dart';
+import 'package:transparent_image/transparent_image.dart';
+
+void main() {
+  runApp(MyApp());
+}
+
+class MyApp extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    final title = 'Desvanecer imágenes';
```

Comment:
Fade in images
En este caso no podemos traducirlo porque hay una imagen que lo muestra en ingles